package.nls.es.json

```
@@ -17,28 +17,28 @@
-    "mql_tools.commands.InsImport.title": "Insertar DLL/EX5 como #import",
-    "mql_tools.commands.InsTime.title": "Insertar fecha y hora como AAAA.MM.DD hh:mm:ss",
-    "mql_tools.commands.InsIcon.title": "Insertar icono como #property",
-    "mql_tools.commands.openInME.title": "Abrir en 'MetaEditor'",
-    "mql_tools.commands.openTradingTerminal.title": "Abrir terminal de trading",
-    "mql_tools.commands.createCommentary.title": "Crear comentario de función",
-    "mql_tools.commands.reportNoisyDiagnostic.title": "Reportar diagnóstico clangd ruidoso",
-    "mql_tools.commands.runBacktest.title": "Ejecutar backtest",
-    "mql_tools.commands.startDebugging.title": "Iniciar sesión de depuración MQL",
-    "mql_tools.commands.stopDebugging.title": "Detener sesión de depuración MQL",
-    "mql_tools.commands.showStartupPage.title": "Mostrar página de bienvenida"
+    "mql_tools.configuration.AutoVersionBump.description": "Automatically bump the minor version in #property version before each compile.","mql_tools.configuration.VersionConstantNames.description": "Names of const string variables to auto-bump on compile (e.g. EA_VERSION).","mql_tools.commands.compileFile.title": "Compilar archivo MQL",
```

Biome (2.4.15)

[error] The key mql_tools.configuration.AutoVersionBump.description was already declared.

(lint/suspicious/noDuplicateObjectKeys)